workflows/iuf/operations/management-nodes-rollout/verify-ncn-images-configs.yaml

```diff
@@ -0,0 +1,88 @@
+#
```

Comment:
`verify-ncn-images-configs.yaml` is now enhanced version of `verify-images-and-configuration.yaml` which is made more modular. With these changes, can `verify-images-and-configuration.yaml` be removed as well??
[https://github.com/Cray-HPE/docs-csm/blob/CASMINST-6467/workflows/iuf/operations/management-nodes-rollout/verify-images-and-configuration.yaml]

Reply:
Yes, I will remove the `verify-images-and-configuration.yaml` file.